### PR TITLE
docs(jsdoc): #3840 document all Meteor Template helpers in 1 @namespace

### PR DESCRIPTION
**Pull request by machikoyasuda**
Resolves #3840  
Impact: **minor**  
Type: **docs**

## Issue
- Meteor Template helpers were not documented, or not displaying on api.docs.reactioncommerce.com

## Solution
- added `@memberof Templates` to methods
- added `@namespace Templates`
- added `@file` information with Blaze helper method docs links

## Breaking changes
none

## Testing
1. Pull this PR
2. Run `npm run docs`
3. Open `file:///tmp/reaction-docs/Templates.html`

## Screenshots

screen shot 2018-02-26 at 4 39 47 pm

### CODE REVIEW

File: client/modules/core/helpers/apps.js [hunk 1]
```diff
@@ -6,44 +6,17 @@ import { Reaction } from "/client/api";
...
- *   label: "Stripe"
- *   name: "reaction-stripe"
- *   packageId: "QqkGsQCDRhg2LSn8J"
- *   priority: 1
- *   provides: "paymentMethod"
- *   template: "stripePaymentForm"
- *   etc: "additional properties as defined in Packages.registry"
- *   ...
- *  }]
+ * @method Apps
```
Comment (zenweasel) [inline client/modules/core/helpers/apps.js]: I don't know that this is actually a template helper? We use it in templates but I think we use it elsewhere.

Comment (machikoyasuda) [reply client/modules/core/helpers/apps.js]: Would `reactionApps` be a better fit in `Helpers`?

**Comment by aldeed** [comment]
Note that I made some changes in https://github.com/reactioncommerce/reaction/pull/4213 that might impact this PR a little. I think it would still be fine to update and merge this PR if time permits, but it's also not critical IMO because we're working toward removing Blaze.

**Comment by aldeed** [comment]
I fixed up and merged this

**Pull request merged**